synthetics/test-config/private-locations.rst

```diff
@@ -115,7 +115,7 @@ Follow these steps to limit logging:
 
 Adding certificates in Synthetics
```

Comment:
```suggestion
Add certificates in Synthetics
```

Reply:
Hi @gschatz-splunk! I can make this change, but it is pre-existing in docs: https://docs.splunk.com/observability/en/synthetics/test-config/private-locations.html#adding-certificates-in-synthetics

Would you still want me to proceed?

Reply:
yep! I added it since i was in there and we had the opportunity to change it ;)